winui/Cartesian-Charts/Line.md

```diff
@@ -256,7 +256,7 @@ this.Content = chart;
 
 ### Spline rendering types
 
-The [SplineType](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SplineSeries.html#Syncfusion_UI_Xaml_Charts_SplineSeries_SplineType) allows you to change the spline curve in series. By default, the value of [SplineType](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SplineSeries.html#Syncfusion_UI_Xaml_Charts_SplineSeries_SplineType) is `Natural`.
+The [Type]() property allows you to change the spline curve in series. By default, the value of [Type]() is `Natural`.
```

Comment:
The [Type]() property allows you to change the spline curve in series. By default, the value of the [Type]() is `Natural.`